firestore/google/cloud/firestore_v1beta1/_helpers.py

```diff
@@ -768,7 +795,7 @@ def get_doc_id(document_pb, expected_prefix):
     return document_id
 
 
-def remove_server_timestamp(document_data):
+def remove_server_timestamp(document_data, top_level=True):
```

Comment:
This is really more of a `process_server_timestamps` that returns a `document_data` without the transforms. We also need a much more specific name than `field_paths` to describe the return type.

Reply:
Done.